front/app/modules/commercial/customizable_navbar/admin/containers/EditNavbarItemForm/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
The form that opens when you go to `admin/settings/navigation/edit-navbar-item:navbarItemId`. Wrapper around the `NavbarItemForm` that handles submitting the form etc.

Reply:
`edit-navbar-item` is a weird url, if that's all the same to you I'd propose changing to `...navabar-items/edit/:navbarItemId`